imports/plugins/core/orders/client/templates/workflow/shippingInvoice.js

```diff
@@ -7,6 +7,7 @@ import { ReactiveVar } from "meteor/reactive-var";
 import { Reaction, i18next, Logger } from "/client/api";
 import { NumericInput } from "/imports/plugins/core/ui/client/components";
 import { Media, Orders, Shops } from "/lib/collections";
+import _ from "lodash";
```

Comment:
we did agree, and it's not documented yet, so I will add now, but the general order of import statements should be:

import npm
import meteor
import local